etc/kayobe/ansible/pulp-docker-registry-login.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Instead of a kayobe hook we can use https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2023.1/etc/kayobe/containers/pulp/post.yml

Reply:
Good to know about these, done now.

Reply:
Need to remove this one?

Reply:
One day I'll handle symlinks correctly first try, seems I just deleted the link twice...
Done now

Reply:
I still get the order wrong every time for `ln -s`